### PR TITLE
[#168693658] Add support for auth routes (assertConsumer and slo)

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "date-fns": "^1.30.1",
-    "fp-ts": "1.12.0",
+    "fp-ts": "^1.0.0",
```
[the remark this answers is not in the document]
Comment (BurnedMarshal) [reply package.json]: Yes, there is a conflict on versions of `fp-ts` used by `italia-ts-commons` (1.12.0) and `io-ts`(^1.0.0 that provide a 1.18.2). Now in this project if we use the version `1.12.0` of `fp-ts` the transcompilation process fail. I suggest resolving the conflict on top of the two libraries if possible.

[the remark this answers is not in the document]
Comment (cloudify) [reply package.json]: `italia-ts-commons` requires `io-ts` `v1.8.5` like this project (see https://github.com/teamdigitale/io-ts-commons/blob/master/package.json#L46)

[the remark this answers is not in the document]
Comment (BurnedMarshal) [reply package.json]: It's working now, thanks. Fixed in 322703a

**Comment by BurnedMarshal** [comment]
@cloudify sorry for the last commit after your approval. It removes code duplication inside the test file.

**Pull request merged**